Buildings/Experimental/DHC/Loads/Examples/BaseClasses/BuildingTimeSeries.mo

```diff
@@ -2,19 +2,21 @@ within Buildings.Experimental.DHC.Loads.Examples.BaseClasses;
 model BuildingTimeSeries
   "Building model with heating and cooling loads provided as time series"
   extends Buildings.Experimental.DHC.Loads.BaseClasses.PartialBuilding(
-    redeclare package Medium=Buildings.Media.Water,
+    redeclare package Medium=MediumW,
     final have_fan=false,
     final have_pum=true,
     final have_eleHea=false,
     final have_eleCoo=false,
     final have_weaBus=false);
-  package Medium2=Buildings.Media.Air
-    "Load side medium";
+  package MediumA=Buildings.Media.Air
```

Comment:
@Helarga @AntoineGautier Remind me about the discussion on the (`Medium1`, `Medium2`) vs (`MediumW`, `MediumA`), did we decide to use (`MediumW` and `MediumA`) in `Example` and `Validation` package, but use (`Medium1` and `Medium2`) in models?

Reply:
@JayHuLBL yes we did, Medium1 "Source", Medium2"Load" in models

Reply:
@JayHuLBL  https://github.com/lbl-srg/modelica-buildings/pull/2296